tony-history-server/src/main/resources/tony-default.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0"?>
```

Comment:
This file, and the `tony-history-server/src/main/resources/log4j.properties` look copied from `tony-core`. Why do you need these here?